### ISSUE TITLE
Fix duplicate modules

### ISSUE THREAD

**Issue by tantaman** [tantaman/strut2#34]
https://github.com/FormidableLabs/inspectpack/#set-resolvealias-in-your-webpack-configuration

```
## main.js
draft-js (Found 2 resolved, 2 installed, 2 depended. Latest 0.11.7.)
  0.10.5 ~/draft-js-modifiers/~/draft-js
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js-checkable-list-item@^3.0.4 -> draft-js-modifiers@~0.2.2 -> draft-js@~0.10.5
  0.11.7 ~/draft-js
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js@~0.11.5

fbjs (Found 2 resolved, 2 installed, 2 depended. Latest 2.0.0.)
  0.8.17 ~/draft-js-modifiers/~/fbjs
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js-checkable-list-item@^3.0.4 -> draft-js-modifiers@~0.2.2 -> draft-js@~0.10.5 -> fbjs@^0.8.15
  2.0.0 ~/fbjs
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js@~0.11.5 -> fbjs@^2.0.0

immutable (Found 2 resolved, 5 installed, 6 depended. Latest 3.8.2.)
  3.7.6 ~/draft-js-checkable-list-item/~/immutable
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js-checkable-list-item@^3.0.4 -> immutable@~3.7.4
  3.7.6 ~/draft-js-modifiers/~/immutable
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js-checkable-list-item@^3.0.4 -> draft-js-modifiers@~0.2.2 -> immutable@~3.7.4
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js-checkable-list-item@^3.0.4 -> draft-js-modifiers@~0.2.2 -> draft-js@~0.10.5 -> immutable@~3.7.4
  3.7.6 ~/draft-js-plugins-editor/~/immutable
    strut2@0.0.0 -> draft-js-plugins-editor@^3.0.0 -> immutable@~3.7.4
  3.7.6 ~/draft-js/~/immutable
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> draft-js@~0.11.5 -> immutable@~3.7.4
  3.8.2 ~/immutable
    strut2@0.0.0 -> draft-js-markdown-shortcuts-plugin@^0.6.1 -> immutable@~3.8.2

* Understanding the report: Need help with the details? See:
  https://github.com/FormidableLabs/inspectpack/#diagnosing-duplicates
* Fixing bundle duplicates: An introductory guide:
  https://github.com/FormidableLabs/inspectpack/#fixing-bundle-duplicates
```

https://formidable.com/blog/2018/finding-webpack-duplicates-with-inspectpack-plugin/

---

tldr: switch to yarn, use their resolutions operator.
